<launch>
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find map_generator)/rviz/display_grid_map_3d.rviz"/>

    <node pkg="map_generator" type="map_generator_2d" name="map_generator_2d" output="screen">
        <param name="init_x" value="0"/>
        <param name="init_y" value="0"/>

        <param name="map/x_size" value="10"/>
        <param name="map/y_size" value="10"/>
        <param name="map/z_size" value="2"/>

        <param name="EllipseObstacle/lower_r" value="0.5"/>
        <param name="EllipseObstacle/upper_r" value="2.5"/>

        <param name="PillarObstacle/lower_h" value="0.3"/>
        <param name="PillarObstacle/upper_h" value="2.5"/>
        <param name="PillarObstacle/lower_w" value="0.1"/>
        <param name="PillarObstacle/upper_w" value="0.8"/>

        <param name="map/circle_number" value="60"/>
        <param name="map/pillar_number" value="150"/>
        <param name="map/resolution" value="0.1"/>

        <param name="sensor/rate" value="0.5"/>
    </node>
</launch>